Bottom: c53fd055ab121287eaecc2bf59a8f09fc8c0ff38
Top:    edad6e8580659bb55a7ef8a1fc4fe1b8895a7d65
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-09-09 12:23:48 +0100

Build with asserts for polygons


---

diff --git a/src/polygon.c b/src/polygon.c
index 48aac13..1e892e2 100644
--- a/src/polygon.c
+++ b/src/polygon.c
@@ -71,6 +71,7 @@ dicer output is used for HIDs which cannot render things with holes
 /* special polygon editing routines
  */
 
+#undef NDEBUG
 #ifdef HAVE_CONFIG_H
 #include "config.h"
 #endif
diff --git a/src/polygon1.c b/src/polygon1.c
index 74cb0cf..ce509d2 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -39,6 +39,7 @@
       are marked
 */
 
+#undef NDEBUG
 #include	<assert.h>
 #include	<stdlib.h>
 #include	<stdio.h>
@@ -102,7 +103,7 @@ int vect_inters2 (Vector A, Vector B, Vector C, Vector D, Vector S1,
 #undef DEBUG_JUMP
 #undef DEBUG_GATHER
 #undef DEBUG_ANGLE
-#undef DEBUG
+#define DEBUG
 #ifdef DEBUG
 #define DEBUGP(...) pcb_fprintf(stderr, ## __VA_ARGS__)
 #else
